applications/subswap/README.md

```diff
@@ -0,0 +1,6 @@
+#Subswap
+
+Contract Deploy doc in [contract](contract/)
+
+Frontend doc in [contract](frontend/)
```

Comment:
1. Frontend doc in [frontend], not [contract]
2. can you directly use url of https://github.com/XinFinOrg/subswap-frontend